packages/ceramic-cli/README.md

```diff
@@ -1,40 +1,98 @@
 # Ceramic CLI
 
-> TODO: description
+> A command line interface that allows you to interact with the Ceramic protocol.
 
 ## Usage
-
-### CLI
 ```
-ceramic daemon
+$ ceramic -h
+```
 
-ceramic create <doctype> <genesis-content> [--onlyGenesis]
+### Not yet implemented commands
 
-ceramic show <docId>
+#### `ceramic pin add <docId>`
+Pin a given document.
 
-ceramic status <docId>
+#### `ceramic pin rm <docId>`
+Unpin a given document.
 
-ceramic watch <docId>
+#### `ceramic pin ls`
+List pinned documents.
 
-ceramic change <docId> <new-content>
 
+#### `ceramic create-from-schema <schema-docId>`
+Interactively create a new document using a given schema. This command will prompt you to enter the information needed in order to create a new Ceramic Tile based on the given schema.
 
-ceramic user did
 
-ceramic user sign <payload | path>
+#### `ceramic user did`
+Show the DID of the user.
 
-ceramic user encrypt <payload>
+#### `ceramic user sign <payload>`
+Ask the user to sign a given payload.
+
+#### `ceramic user encrypt <payload>`
+Ask the user to encrypt a given payload.
+
+#### `ceramic user decrypt <JWE | CWE>`
+Ask the user to decrypt a given JWE or CWE.
 
-ceramic user decrypt <JWE | CWE><Paste>
-```
 
-### HTTP API
+## Ceramic http api
+The Ceramic daemon creates an http endpoint from which various operations can be performed.
+
+### `/api/v0/create` - `POST`
+Create a new Ceramic document.
+
+#### Post data
+Only `content` is required.
+
+```js
+{
+  content: <content-json>,
+  doctype: <string>, // defaults to "tile"
+  owners: <owners-string-array> // defaults to current user
+}
 ```
-/api/v0/create - POST
 
-/api/v0/show/<document-id> - GET
+### `/api/v0/show/<document-id>` - `GET`
+Returns the content of the given document.
+
+##### Optional query parameters:
+* `?version=<number>` - the version of the document to show
```

Comment:
@oed how will we implement versions? Based on numbers as you have here, or would it be better to request the CID of the given anchor record? Or does the protocol automatically add numbers to the records so they can be queried by either?

Reply:
So this needs to be documented in the spec. The number refers to the n:th sequential anchor record. So essentially:
* 0: the genesis record
* 1: the first anchor record
* 2: the second anchor record
* n: the n:th anchor record